plugin-server/src/cdp/templates/_transformations/bot-detection/bot-detection.template.test.ts

```diff
@@ -2,6 +2,13 @@ import { HogFunctionInvocationGlobals } from '../../../types'
 import { TemplateTester } from '../../test/test-helpers'
 import { template } from './bot-detection.template'
 
+// TODO we shouldn't need these, our invocation code should use the template defaults correctly
```

Comment:
I didn't want to touch this problem in this PR. (I didn't create it, the problem already existed, I just added a comment to call it out)